spring-cloud-kubernetes-core/src/main/java/org/springframework/cloud/kubernetes/Constants.java

```diff
@@ -0,0 +1,9 @@
+package org.springframework.cloud.kubernetes;
+
+public final class Constants {
+
+	public static final String SPRING_CLOUD_KUBERNETES_CLIENT_PROPS = "spring.cloud.kubernetes.client";
```

Comment:
No need for a separate class is there? I'd assume keep it in  the properties class.

Reply:
Fixed